lib/LedgerSMB/Tax.pm

```diff
@@ -47,7 +47,7 @@ sub init_taxes {
     my @accounts = split / /, $taxaccounts;
     if ( defined $taxaccounts2 ) {
         #my @tmpaccounts = @accounts;#unused var
-        $#accounts = -1;# empty @accounts,@accounts=();
+        @accounts=(); # empty @accounts
```

Comment:
I think the replacement code (while I still think that `-1` isn't magical) is better than the original. Lets leave this one in.